src/shark/clients/consent_changes_client.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
+
+import Client from 'src/shark/client'
+
+class ConsentChangesClient {
```

Comment:
why do we need separate client to this? 🤔 aren't we using the same api in the end?

Reply:
Wish of the masses I think. We have a client for each resource. Consent changes are technically a resource.